.github/workflows/test.yml

```
@@ -22,7 +22,23 @@ jobs:
     runs-on: ${{ matrix.runs-on }}
     steps:
       - uses: actions/checkout@v2
-      - uses: ./
+      - name: Test Action - local 
+        if: ${{ env.ACT }}
+        uses: ./actiontest
```

this would need to change to: 

```suggestion
        uses: ./
```